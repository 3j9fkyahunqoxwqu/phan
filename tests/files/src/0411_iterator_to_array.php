<?php

function to_array411(iterable $input): array {
    if (is_array($input)) {
        return iterator_to_array($input);
    } else {
        return iterator_to_array($input, $preserve_keys = true);
    }
}
var_export(iterator_to_array('string'));  // should warn. But the above uses are valid.
