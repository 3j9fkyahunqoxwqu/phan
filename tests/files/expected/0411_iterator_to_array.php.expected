%s:10 PhanTypeMismatchArgumentInternal Argument 1 (it) is string but \iterator_to_array() takes iterable
